### PR TITLE
Address a TODO in MPS testing now that it's possible

### PR DESCRIPTION
**Pull request by aslvrstn**

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -7874,11 +7861,7 @@ def test_numpy_ref_mps(self, device, dtype, op):
         for sample_input in inputs:
             self.compare_with_reference(op, op.ref, sample_input)
 
-# TODO: Actually instantiate that test for the "mps" device to better reflect what it is doing.
-# This requires mps to be properly registered in the device generic test framework which is not the
-# case right now. We can probably use `allow_mps` introduced in https://github.com/pytorch/pytorch/pull/87342
-# to achieve this.
-instantiate_device_type_tests(TestConsistency, globals(), only_for="cpu")
+instantiate_device_type_tests(TestConsistency, globals(), allow_mps=True)
```
Comment (mruberry) [inline test/test_mps.py:7864]: `only_for="mps"`?

Comment (aslvrstn) [reply test/test_mps.py:7864]: Ha, of course, thank you!

File: test/test_mps.py [hunk 2]
```diff
@@ -7745,17 +7743,7 @@ def get_samples():
                 cpu_out = op(*cpu_args, **cpu_kwargs)
                 mps_out = op(*mps_args, **mps_kwargs)
 
-                if op.name == "nn.functional.conv2d" and dtype == torch.float32:
-                    atol = 1e-4
```
Comment (kulinseth) [inline test/test_mps.py:7749]: We don't need these tolerate values anymore ?

**Review by kulinseth — APPROVED** [review]
Overall looks good. Not clear about the rtol  and atol changes if they are no longer needed ?

File: test/test_mps.py (hunk 2 above)
Comment (aslvrstn) [reply test/test_mps.py:7749]: I moved them over into the function metadata in [torch/testing/_internal/common_methods_invocations.py](https://github.com/pytorch/pytorch/pull/87529/files#diff-18a0c3e7cd07089b34760028c387d39de1aa393172f1d5f123692106b3a3d144)

At least I think I did, and tests look happy.

**Pull request closed without merging**